penguinswim/build.gradle

```diff
@@ -18,11 +18,7 @@ apply plugin: 'com.android.dynamic-feature'
 
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
-
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
Do we want to keep the Java 8 source compatibility?

Reply:
It is 1.8 by default now (if not specified), so it is safe to remote it.